src/server/req_stat.c

```diff
@@ -923,16 +887,12 @@ void
 update_license_ct(attribute *pattr, char *buf)
 {
 	buf[0] = '\0';
-	if (licstate_is_up(LIC_SERVER))
-		print_license_ct(LIC_SERVER, buf);
-	else if (licstate_is_up(LIC_NODES))
-		print_license_ct(LIC_NODES, buf);
-	else if (licstate_is_up(LIC_SOCKETS))
-		print_license_ct(LIC_SOCKETS, buf);
-	else if (licenses.lb_trial == 1)
-		print_license_ct(LIC_TRIAL, buf);
-	else
-		print_license_ct(last_valid_attempt, buf);
+	if (licstate_is_up(LIC_NODES) || licstate_is_up(LIC_SOCKETS)) {
+		sprintf(buf, "Avail_Global:%d Avail_Local:%d Used:%d High_Use:%d",
+				licenses.lb_glob_floating, licenses.lb_aval_floating,
+				licenses.lb_used_floating, licenses.lb_high_used_floating);
+	} else
```

Comment:
Do we need this else case as numbers will be zero anyway if things are not configured?